app/components/ui/Navigation/Navigation.jsx

```diff
@@ -86,7 +87,7 @@ class Navigation extends React.Component {
               )
             }
           </div>
-          {!isSFFamiliesSite()
+          {title !== 'SF Families'
```

Comment:
showMobileNav or something similar would be the configurable property in this case -- true for everyone but sffamilies.

Reply:
We can add a new label, will wait for other reviews too.